logstreams/src/main/java/io/zeebe/distributedlog/restore/log/RestoreClientFactory.java

```diff
@@ -13,16 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.zeebe.distributedlog.restore;
+package io.zeebe.distributedlog.restore.log;
```

Comment:
This is not only for logstream, but also for restoring snapshot. So why do you have to move to package 'log'?

Reply:
I moved the logged event replication stuff specifically to this package, since it's sort of a self contained feature (i.e. replicating logged events). Admittedly the name is poorly chosen, but it was more as a means to organise things. No strong argument there other than its a personal preference, I find it easier to navigate things this way.